dev-postgres-powersync/powersync-services/powersync.yaml

```diff
@@ -44,21 +47,8 @@ services:
       # or e.g.: Via a command line parameter
       #     command: ['start', '-r', 'unified', '-sync64', '[base64 encoded content]']
 
-      # The following will be inserted into the powersync.yaml file via templating
-      # Templates are used for filesystem and base64 configuration files
-      PS_PG_URI: postgres://${PG_DATABASE_USER}:${PG_DATABASE_PASSWORD}@db:${PG_DATABASE_PORT}/${PG_DATABASE_NAME}
       PS_MONGO_URI: mongodb://mongo:27017/powersync_wger
-
-      # Note that powersync.yaml->client_auth->allow_local_jwks must be true for local URLs to work
-      PS_JWKS_URL: ${PS_JWKS_URL}
-
-      # The port which the PowerSync API server should run on
-      PS_PORT: ${PS_PORT}
-
-      # Potential JWKs public key template
-      # PS_JWK_N:
-      # PS_JWK_E:
-      # PS_JWK_KID:
+      PS_JWKS_URL: http://web:8000/api/v2/powersync-keys
 
     ports:
-      - ${PS_PORT}:${PS_PORT}
+      - 8080:8080
```

Comment:
why are we now hardcoding this?

Reply:
all other services have hardcoded ports so this doen't really matter. We can also add the ps_port and add a default value if we come to need this level of configurability